README.md

```diff
@@ -9,6 +9,31 @@ for production use.**
 **One place for service teams to find styles, components and patterns for
 designing government services.**
 
+## GOV.UK Frontend packages
+
+Design System consumes [GOV.UK Frontend](https://github.com/alphagov/govuk-frontend) packages via [NPM](https://www.npmjs.com/).
+These are defined in the [package.json](package.json) file.
+
+---------------------
+**NOTE:**
+For the time being we are consuming private packages. To access private packages, you will first need to log in to NPM with
+
+`npm login`
+
+--------------------
+
+To install all currently used packages for the first time, navigate to the application root folder and run:
+
+`npm install`
+
+To install a new GOV.UK Frontend package type:
```

Comment:
I worry that this is here because of some badly-written acceptance criteria on my part. I wrote

> 'Documentation exists to tell developers how to install the dependencies when working with it locally'

when really I meant

> 'Documentation exists to tell developers that they'll need to login in order to install the dependencies'

Or do we think it is useful to tell people how to add new dependencies?

Reply:
no necessary, most devs should know. 
shall we take it out @alex-ju ?

Reply:
We should. At least for now.